src/App.js

```diff
@@ -0,0 +1,91 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+
+<nav>
```

Comment:
code should be formatted properly - with proper indentations for nested elements.